sale_layout_multi_company/views/sale_layout_view.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="report_configuration_form_view_company" model="ir.ui.view">
+        <field name="name">report_configuration_form_view_company</field>
+        <field name="model">sale.layout_category</field>
+        <field name="inherit_id" ref="sale.report_configuration_form_view"/>
+        <field name="arch" type="xml">
+            <field name="sequence" position="after">
+                <field name="company_id" groups="base.group_multi_company" widget="selection"/>
```

Comment:
I miss here `company_ids` field, which is the main aim of the base_multi_company module.

Reply:
@pedrobaeza thanks for the review, done!